src/test/java/org/apache/maven/shared/utils/xml/XmlWriterUtilTest.java

```diff
@@ -86,7 +75,6 @@ public void testWriteLineBreakXMLWriterInt()
     {
         XmlWriterUtil.writeLineBreak( xmlWriter, 10 );
         writer.close();
-        assertTrue( StringUtils.countMatches( output.toString(), XmlWriterUtil.LS ) == 10 );
```

Comment:
Did you remove the assertion on purpose? If so, why?

Reply:
nope, just an error. I'll fix it.

Reply:
done